### PR TITLE
taskwarrior: update to 3.0.2

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

- taskwarrior: update to 3.0.2
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>

Package(s) Affected
-------------------

- taskwarrior: 3.0.2

Security Update?
----------------

No

Build Order
-----------

```
#buildit taskwarrior
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

**Comment by OriginCode** [comment]
LGTM on amd64:
image

File: app-productivity/taskwarrior/autobuild/templates [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Template: taskwarrior/migrate_from_2
+Type: note
+Description: Upgrading from Taskwarrior 2 Needs Manual Migration
```
Comment (MingcongBai) [inline app-productivity/taskwarrior/autobuild/templates]: Migration Notes for Taskwarrior 2 Users

File: app-productivity/taskwarrior/autobuild/templates [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ Taskwarrior 3 changed the way it stores and synchronises tasks, you would need
+ to export all your previous tasks before upgrading using the following command:
```
Comment (MingcongBai) [inline app-productivity/taskwarrior/autobuild/templates]: Taskwarrior 3 introduced a new format for storing and synchronising tasks. Before upgrading Taskwarrior, you must export all your previous data using the following command:

File: app-productivity/taskwarrior/autobuild/templates [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+   
+ $ task export > all-tasks.json
```
Comment (MingcongBai) [inline app-productivity/taskwarrior/autobuild/templates:7]:
```
    task export > all-tasks.json
```

With indentation

File: app-productivity/taskwarrior/autobuild/templates [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+   
+ After upgrading to Taskwarrior 3, re-import your tasks using the following
+ command:
+  
+ $ task import rc.hooks=0 all-tasks.json
```
Comment (MingcongBai) [inline app-productivity/taskwarrior/autobuild/templates:31]: Ditto above about command formatting.

File: app-productivity/taskwarrior/autobuild/templates [hunk 5]
```diff
@@ -0,0 +1,33 @@
+Template: taskwarrior/migrate_from_2
+Type: note
+Description: Upgrading from Taskwarrior 2 Needs Manual Migration
+ Taskwarrior 3 changed the way it stores and synchronises tasks, you would need
+ to export all your previous tasks before upgrading using the following command:
+
```
Comment (MingcongBai) [inline app-productivity/taskwarrior/autobuild/templates]: You'd need to add a `.` to indicate new line.

File: app-productivity/taskwarrior/autobuild/templates [hunk 6]
```diff
@@ -0,0 +1,33 @@
+Template: taskwarrior/migrate_from_2
+Type: note
+Description: Upgrading from Taskwarrior 2 Needs Manual Migration
+ Taskwarrior 3 changed the way it stores and synchronises tasks, you would need
+ to export all your previous tasks before upgrading using the following command:
+   
+ $ task export > all-tasks.json
+   
+ After upgrading to Taskwarrior 3, re-import your tasks using the following
```
Comment (MingcongBai) [inline app-productivity/taskwarrior/autobuild/templates:28]: Split this into another template.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-productivity/taskwarrior/autobuild/templates (hunk 3 above)
Comment (MingcongBai) [reply app-productivity/taskwarrior/autobuild/templates:7]: If you have not yet exported your data, please select "No." After you have finished exporting your data, return and select "Yes" to proceed.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**